servers/tests/org.fusesource.ide.server.karaf.core.tests/src/test/java/org/fusesource/ide/server/karaf/core/server/subsystems/KarafUtilsTest.java

```diff
@@ -37,6 +39,13 @@
 	public TemporaryFolder testFolder = new TemporaryFolder();
 	
 	@Test
+	public void testGetBundleSymbolicNameWithoutParameters() throws CoreException {
+		assertThat(KarafUtils.stripParametersFromSymbolicName("mySampleApp;blueprint.aries.xml-validation:=false").indexOf(';')).isEqualTo(-1);
+		BundleDetails bd = new BundleDetails("mySampleApp;blueprint.aries.xml-validation:=false", "1.0.0");
+		assertThat(bd.getSymbolicName().indexOf(';')).isEqualTo(-1);
```

Comment:
why not checking assertThat(bd.getSymbolicName()).isEqualTo("mySampleApp"); ?
would be more precise test and easier to read, what do you think?